src/components/form/SelectCollateral/index.tsx

```diff
@@ -13,12 +13,14 @@ export interface SelectCollateralProps {
   formMethods: FormContextValues<SplitPositionFormMethods>
   splitFromCollateral: boolean
   tokens: Token[]
+  setAllowanceError: any
```

Comment:
Add a proper type here

Reply:
Mmmm I think with `Maybe<Error>` should be OK, please check it before commit

Reply:
Replaced setAllowationError to `cleanAllowationError` callback